src/devices/cpu/m6809/m6809.lst

```diff
@@ -600,6 +646,113 @@ TFR:
 	@dummy_vma(hd6309_native_mode() ? 2 : 4);
 	return;
 
+FREERUN:
+	// A CPU test mode that increments the address bus until reset.
+	m_free_run = true;
+	return;
+
+X18:
+	@m_temp.b.l = read_opcode_arg(m_pc.w);
+	m_temp.b.l = (m_cc & m_temp.b.l) << 1;
+	m_cc = m_temp.b.l | ((m_cc & 0x04) >> 1);
+	return;
+
+XANDCC:
+	@eat(1);
+	goto ANDCC;
+
+XRES:
+	set_ea(VECTOR_RESET_FFFE);
+	standard_irq_callback(M6809_SWI);
+	set_regop16(m_s);
+	m_temp.w = entire_state_registers();
+	@dummy_read_opcode_arg(0);
+	@dummy_vma(1);
+	%PUSH_REGISTERS;
+	goto INTERRUPT_VECTOR;
+
+XNC8:
+	if (m_cc & CC_C)
+		goto COM8;
+	else
+		goto NEG8;
+
+XDEC8:
+	@m_temp.b.l = read_operand();
+	if (m_temp.b.l)
+		m_cc |= CC_C;
+	else
+		m_cc &= ~CC_C;
+
+	m_temp.b.l = set_flags<uint8_t>(CC_NZ, m_temp.b.l, 1, m_temp.b.l - 1);
+	if(!hd6309_native_mode() || !is_register_addressing_mode())
+	{
+		@dummy_read_opcode_arg(0);
+		;
+	}
+	@write_operand(m_temp.b.l);
+	return;
+
+XCLR8:
+	@read_operand();
+	m_cc &= ~CC_NZV;
+	m_cc |= CC_Z;
+	if(!hd6309_native_mode() || !is_register_addressing_mode())
+	{
+		@dummy_read_opcode_arg(0);
+		;
+	}
+	@write_operand(0);
+	return;
+
+XST8:
+	// ToDo: Flag setting is much more complex see:
+	// https://github.com/hoglet67/6809Decoder/wiki/Undocumented-6809-Behaviours#store-immediate
```

Comment:
Please put `TODO` in all caps.  It’s recognised by various text editors and tools.

Reply:
Addressed. Committed.